charts/pega/charts/hazelcast/templates/clustering-service-migration.yaml

```diff
@@ -0,0 +1,137 @@
+{{ if and (and (eq (include "performDeployment" .) "true") (eq (include "isMigrationInit" .) "true"))
+   (eq (include "isClusteringServiceEnabled" .) "true") }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "clusteringServiceName" . }}-migration-job
+  namespace: {{ .Release.Namespace }}
+spec:
+  backoffLimit: 0
+  template:
+    metadata:
+      name: {{ template "clusteringServiceName" . }}-migration-job
+    spec:
+      serviceAccountName: {{ template "clusteringServiceName" . }}-migration-sa
+      containers:
+        - name: migration-job
+          image: {{ .Values.migratorImage }}
+          command:
+            - bin/bash
+            - -c
+            - |
+              {{ range $index, $dep := .Values.global.tier}}
+              {{ if (eq $dep.nodeType "WebUser") }}
+              labelName="pega-{{ $dep.name }}"
```

Comment:
is there a possibility for an env to not have webtier?

Reply:
@nanil4568 , updated to take the first pega tier